orchestrator/crates/orchestrator/src/main.rs

```diff
@@ -17,7 +22,8 @@ static A: jemallocator::Jemalloc = jemallocator::Jemalloc;
 #[allow(clippy::needless_return)]
 async fn main() {
```

Comment:
Consider returning the never type instead.
```suggestion
async fn main() -> ! {
```

Reply:
intresting will look into this in details